src/components/NewGalaxyModal.js

```diff
@@ -186,59 +198,118 @@ export const NewGalaxyModal = _.flow(
-        ['STARTING', () => `Cloud environment is starting. ${waitMessage}`],
-        ['RUNNING', () => 'Environment consists of an application and cloud compute.'],
+        ['STOPPED', () => `is paused.`],
+        ['PRESTOPPING', () => 'is preparing to pause.'],
+        ['STOPPING', () => `is pausing. ${waitMessage}`],
+        ['PRESTARTING', () => 'is preparing to start.'],
+        ['STARTING', () => `is resuming. ${waitMessage}`],
+        ['RUNNING', () => 'consists of application configuration, cloud compute and persistent disk(s).'],
         ['ERROR', () => `An error has occurred on your cloud environment.`]
       )
   }
 
+  // TODO Refactor this and the duplicate in NewRuntimeModal.js
+  const renderGalaxyCostBreakdown = (kubernetesRuntimeConfig, dataDiskSize) => {
+    const runningComputeCost = getGalaxyComputeCost({ status: 'RUNNING', kubernetesRuntimeConfig })
+    const pausedComputeCost = getGalaxyComputeCost({ status: 'STOPPED', kubernetesRuntimeConfig })
+
+    return div({
+      style: {
+        backgroundColor: colors.accent(0.2),
+        display: 'flex',
+        borderRadius: 5,
+        padding: '0.5rem 1rem',
+        marginTop: '1rem'
+      }
+    }, [
+      _.map(({ cost, label, unitLabel }) => {
+        return div({ key: label, style: { flex: 1, ...styles.label } }, [
+          div({ style: { fontSize: 10 } }, [label]),
+          div({ style: { color: colors.accent(), marginTop: '0.25rem' } }, [
+            span({ style: { fontSize: 20 } }, [cost]),
+            span([' ', unitLabel])
+          ])
+        ])
+      }, [
+        { label: 'Running cloud compute cost', cost: Utils.formatUSD(runningComputeCost), unitLabel: 'per hr' },
+        { label: 'Paused cloud compute cost', cost: Utils.formatUSD(pausedComputeCost), unitLabel: 'per hr' },
+        { label: 'Persistent disk cost', cost: Utils.formatUSD(getGalaxyDiskCost(dataDiskSize)), unitLabel: 'per month' }
+      ])
+    ])
+  }
+
+  const renderCloudComputeProfileSection = () => {
+    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '2rem', alignItems: 'center' }
+    return div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+      div({ style: styles.headerText }, ['Cloud compute profile']),
+      div({ style: { ...gridStyle, marginTop: '0.75rem' } }, [
+        h(MachineSelector, { value: kubernetesRuntimeConfig, onChange: v => setKubernetesRuntimeConfig(v) })
+      ])
+    ])
+  }
+
+  const renderPersistentDiskSection = () => {
+    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' }
+    return div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+      div({ style: styles.headerText }, ['Persistent disk']),
+      div({ style: { marginTop: '0.5rem' } }, [
+        'Stores your analysis data '
```

Comment:
```suggestion
        'Stores your analysis data.'
```

Reply:
Wondering if having the documentation about persistent disks linked here (like it is in the cloud env version) would be beneficial? ie:
`Stores your analysis data. Learn more about Persistent disks and where your disk is mounted`